package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "SmartKickers",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Remove this file, because package-lock.json is needless on root in your case.

Reply:
Done